tinyquery/tinyquery.py

```diff
@@ -143,7 +143,7 @@ def flatten_row(output, row, schema, prefix='', ever_repeated=False):
             return output
 
         def process_row(row):
-            for (key, value) in row.iteritems():
+            for (key, value) in row.items():
```

Comment:
This is the one place we iterate over a row, right?  So this could be big?

Reply:
It's iterating over the columns in a row, so should be similar in size to any other column iterators